papers/core_bench.md

```diff
@@ -0,0 +1,98 @@
+I chose Siegel et al.'s paper [@siegel2024corebenchfosteringcredibilitypublished] for our journal club because:
+
+- I really like what these folks are proposing about agents' evaluation.
+- I think that the evaluation of agents is the current bottleneck in the development of new agents.
+- They write some really interesting statements such as *"Before agents can automate scientific research, they must be able to reproduce existing results."*.
+- One of the conclusions is really interesting for one of the projects that I am working on.
+
+## Prior work
+
+With the appearance of the LLMs, agents have gained a lot of importance, especially during the last year. However, the evaluation of these agents is still a challenge, and a bunch of benchmarks and different evaluations have been proposed to evaluate them.
+
+### SWE-Bench
+
+In this work by Jimenez et al. [@jimenez2024swebenchlanguagemodelsresolve], they introduced another agent benchmark. The tasks that they evaluate are based on GitHub Pull Requests extracted from some popular repositories. They evaluate the agents in solving the issues that are opened in these PRs. Nowadays, it is one of the most used benchmarks for evaluating new LLMs.
+
+### AI Agents That Matter
+
+This is previous work from the same authors of the CORE-Bench. In this work, Kappor et al. [@kapoor2024aiagentsmatter] propose different case studies, each with a different evaluation method for some of the more common NLP benchmarks. The particularity is that these evaluations are thought to evaluate more than only the agents' final answers.
+
+### LAB-Bench
+
+This benchmark proposed by Laurent et al. [@laurent2024labbenchmeasuringcapabilitieslanguage] is focused on evaluating agents in the scientific domain, specifically in the medical/biochemistry domain. It is multimodal, including tasks such as analyzing lab images, scientific tables, lab protocols, or retrieving data from biological databases.
+
+![Figure taken from Laurent et al. paper [@laurent2024labbenchmeasuringcapabilitieslanguage] illustrating the results of the different leading LLMs in LAB-Bench.](core_bench_papers/core_bench_2.png)
+
+## Problem setting
+
+- Evaluating LLM-powered agents is a real challenge.
+- It is possible to evaluate only the final result, but interesting conclusions can be lost in the reasoning process.
+- This reasoning evaluations is really important within the develpment of new agents since it can help to understand the agents' limitations.
+- The current evaluation of the agents is a bottleneck in the development of new agents.
+
+## Approach
+
+They proposed a new benchmark that allows the agents to be evaluated more deeply. They propose to evaluate the agents in three different tasks: CORE-Bench-Easy, CORE-Bench-Medium, and CORE-Bench-Hard. The first one is a simple task, the second one is a medium task, and the last one is a hard task. The evaluation is done so that the agents have to provide the answer and the reasoning behind it.
```

Comment:
I think what would have been good is to be more specific about what enables the benchmark to do a deep evaluation.